SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -191,7 +193,7 @@ public IActionResult AskForPostcode_Get(string reference)
         [HttpPost("postcode/{reference}")]
         public IActionResult AskForPostcode_Post(AskForPostcodeViewModel viewModel)
         {
-            if (!PostcodesIoApi.IsValidPostcode(viewModel.Postcode))
+            if (viewModel.Postcode is not null && !PostcodesIoApi.IsValidPostcode(viewModel.Postcode))

```

Comment:
Otherwise we show that an empty postcode is not valid too, when we already have the postcode missing error